scripts/scil_flip_surface.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Script to flip and reverse cortical surface (vtk or freesurfer).
```

Comment:
Remove cortical.
Maybe two sentences, one about flip and one about inverting normal just so it is clearer for the user.

Mentioning what is the use case for inverting the normal would be useful too,